### PR TITLE
fixes #283 and #248

### LINKED ISSUE #283: updated recommendations list file
with new welcome message and fix for linux last tab keyboard shortcut (re #248)

### PR DESCRIPTION
**Pull request by raymak**

### CODE REVIEW

File: phase2/addon/data/fx-guide-recs.json [hunk 1]
```diff
@@ -0,0 +1,892 @@
+[
+  {
+    "id": "welcome",
+    "priority": 0,
+    "trigBehavior": "",
+    "delivContext": "*",
+    "featUseBehavior": "",
+    "presentationData": {
+      "*": {
+        "message": "Here you will see awesome Firefox tips that are just for you. Click the lightbulb icon to see your most recent recommendation.",
```
Comment (raymak) [inline phase2/addon/data/fx-guide-recs.json]: new welcome message

File: phase2/addon/data/fx-guide-recs.json [hunk 2]
```diff
@@ -0,0 +1,892 @@
...
+    "respCommandMap": {
+      "doorhanger": {
+        "primaryButton": "open url -l https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_windows-tabs"
+      }
+    },
+    "tags": "",
+    "platform": "winnt; darwin"
+  },
+  {
+    "id": "lastTabShortcut-linux",
```
Comment (raymak) [inline phase2/addon/data/fx-guide-recs.json]: last tab shortcut for Linux

**Comment by gregglind** [comment]
(it helps for future to also remind me of the those two bugs are in the close comment :) )

File: phase2/addon/data/fx-guide-recs.json (hunk 1 above)
Comment (gregglind) [inline phase2/addon/data/fx-guide-recs.json]: r+

File: phase2/addon/package.json [hunk 3]
```diff
@@ -81,7 +81,7 @@
       "type": "string",
       "title": "Feature recommendation list",
       "description": "Address of the feature recommendations list to load.",
-      "value": "cfr-moz-recs.json",
+      "value": "fx-guide-recs.json",
```
Comment (gregglind) [inline phase2/addon/package.json]: r+

**Comment by raymak** [comment]
superceded by #287

**Pull request closed without merging**